catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -137,19 +147,12 @@ public Metacard read(InputStream inputStream) throws CatalogTransformerException
         for (SaxEventHandler eventHandler : eventHandlers) {
             List<Attribute> attributes = eventHandler.getAttributes();
             for (Attribute attribute : attributes) {
-                Attribute tmpAttr;
 
                 /*
-                 * If metacard already has values in the attribute, put them together into a multivalued list,
+                 * If metacard already has values in the attribute, skip the attribute,
                  * instead of simply overwriting the existing values.
                  */
-                if ((tmpAttr = metacard.getAttribute(attribute.getName())) != null) {
```

Comment:
I thought some more about this. What if you check to see if the attribute is supposed to be multivalued or not? It should be one of the fields in the `AttributeDescriptor`. Then if it is, you can use all the values, and if it's not, skip the values, the way you are here.